doc/source/whatsnew/v0.18.1.txt

```diff
@@ -18,8 +18,7 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
-
-
+- ``pd.crosstab()`` now has ``normalize`` argument for normalizing frequency tables (:issue:`12569`).
```

Comment:
I think you can add a sub-section here with an example / motivation (and add the same example to reshape.rst, with a pointer from here)